src/main/java/com/direwolf20/buildinggadgets/common/items/gadgets/GadgetExchanger.java

```diff
@@ -192,7 +162,7 @@ private boolean exchange(EntityPlayer player, ItemStack stack) {
             BlockPos startBlock = lookingAt.getBlockPos();
             EnumFacing sideHit = lookingAt.sideHit;
 //            IBlockState setBlock = getToolBlock(stack);
```

Comment:
If it's not being used then remove it or add a comment explaining why it's not currently being used

Reply:
Never touched this line in the PR, and this PR is big enough now. Plus I have no idea either why is it here.